shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-suite/src/test/java/org/apache/shardingsphere/test/integration/engine/ddl/BaseDDLIT.java

```diff
@@ -59,6 +61,14 @@ public final void init() throws Exception {
         try (Connection connection = getTargetDataSource().getConnection()) {
             executeInitSQLs(connection);
         }
+        intervalTime();
+    }
+    
+    @SneakyThrows(InterruptedException.class)
+    private void intervalTime() {
+        if ("Cluster".equalsIgnoreCase(getMode())) {
+            TimeUnit.MILLISECONDS.sleep(200);
```

Comment:
long value should use `L`, please change it to `200L`

Reply:
I will fix it.